media/src/main/java/io/wcm/handler/media/ui/ResourceMedia.java

```diff
@@ -70,8 +82,55 @@
   @RequestAttribute(injectionStrategy = InjectionStrategy.OPTIONAL)
   private String cssClass;
 
+  /**
+   * Defines responsive rendition widths for image.
+   * To be used together with 'imageSizes' property. <br>
+   * Example: "{@literal 2560:false,1920:false,1280:false,640:false,320:false}" <br>
```

Comment:
proposal for a better solution: we should not introduce a new syntax to mark sitzes as required or not. we should use the same syntax as used for the "width" property as described here:
https://wcm.io/handler/media/component-properties.html
number with "?" suffix = optional, number without suffix = mandatory

Reply:
good idea. I changed that and moved the parseWidths method to an internal util class.